fixtures/tests/_fixtures/test_monkeypatch.py

```diff
@@ -72,14 +83,112 @@ def test_delete_missing_attribute(self):
             fixture.cleanUp()
             self.assertFalse('new_attr' in globals())
 
+    def _check_static_or_class_method(self, name, new_method):
```

Comment:
so this patches out name with new_method and makes sure that name was callable before, and is callable after, on both class and instances?